docs/integrations/sources/surveymonkey.md

```diff
@@ -1,72 +1,100 @@
+1. In the left navigation bar, click **Sources**. Click **SurveyMonkey**, which takes you to the `Create a source` page.
+1. Click **Authenticate your account**.
+1. Log in to the SurveyMonkey account to authorize your app.
+1. In the `Start Date` field, set the required start date.
+1. Click **Set up source**.
+
+</TabItem>
 
-1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
-3. On the source setup page, select **SurveyMonkey** from the Source type dropdown and enter a name for this connector.
-4. lick `Authenticate your account`.
-5. Log in and Authorize to the SurveyMonkey account
-6. Choose required Start date
-7. click `Set up source`.
 <!-- /env:cloud -->
 
+<TabItem value="oss" label="Airbyte Open Source">
+
 <!-- env:oss -->
 
-**For Airbyte Open Source:**
+1. Go to your [local Airbyte](http://localhost:8000/) page.
+1. In the left navigation bar, click **Sources**. Click **SurveyMonkey**, which takes you to the `Create a source` page.
+1. In the `Access Token` field, enter your SurveyMonkey access token from Step 1.
+1. In the `Start Date` field, set the required start date.
+1. Click **Set up source**.
 
-1. Go to local Airbyte page.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
-3. On the source setup page, select **SurveyMonkey** from the Source type dropdown and enter a name for this connector.
-4. Add **Access Token**
-5. Choose required Start date
-6. Click `Set up source`.
 <!-- /env:oss -->
 
+</TabItem>
+</Tabs>
+
 ## Supported streams and sync modes
 
+Airbyte supports the following SurveyMonkey streams and sync modes. `Incremental` denotes partial support:
+
 - [Surveys](https://api.surveymonkey.com/v3/docs?shell#api-endpoints-get-surveys) \(Incremental\)
 - [SurveyPages](https://api.surveymonkey.com/v3/docs?shell#api-endpoints-get-surveys-survey_id-pages)
 - [SurveyQuestions](https://api.surveymonkey.com/v3/docs?shell#api-endpoints-get-surveys-survey_id-pages-page_id-questions)
-- [SurveyResponses](https://api.surveymonkey.com/v3/docs?shell#api-endpoints-get-surveys-id-responses-bulk) \(Incremental\)
+- [SurveyResponses](https://api.surveymonkey.com/v3/docs?shell#api-endpoints-get-surveys-survey_id-responses-bulk) \(Incremental\)
 - [SurveyCollectors](https://api.surveymonkey.com/v3/docs?shell#api-endpoints-get-surveys-survey_id-collectors)
 - [Collectors](https://api.surveymonkey.com/v3/docs?shell#api-endpoints-get-collectors-collector_id-)
 
 ### Performance considerations
 
-The SurveyMonkey API applies heavy API quotas for default private apps, which have the following limits:
+The SurveyMonkey API applies API quotas for private apps. The following are the default limits:
 
 - 125 requests per minute
 - 500 requests per day
 
-To cover more data from this source we use caching.
+To cover more data from this source, Airbyte uses caching.
```

Comment:
Not sure what this sentence is trying to say, so I'd follow up with an SME on improving this sentence, if needed.